### ISSUE TITLE
Run tests on all supported versions

### ISSUE THREAD

**Issue by Narretz** [angular-ui/ui-select#1431]
The bower.json requires angular >= 1.2.18, which implies that ui-select will run on all versions equal or higher than that. But the tests are only run for the latest version that fulfills this, so atm this is 1.5.0. The test suite should instead run the tests for the latest patch versions of 1.2, 1.3, 1.4 and 1.5.

**Comment by gaetansenn** [comment]
Yes right. I got the same problem when I upgraded to the 1.5 version of angularJS.
